examples/README.md

```diff
@@ -25,15 +25,15 @@ The following examples are included:
 IGNITE_SOURCES=/path/to/ignite3-sources-dir
 ```
 
-3. Build the Ignite Docker image. As a result the `apacheignite/ignite3` image will be built and loaded into the Docker Engine:
+3. Build the Ignite Docker image. As a result the `apacheignite/ignite` image will be built and loaded into the Docker Engine:
```

Comment:
You don't have to build it yourself, let's remove this step - we publish the image as part of the release.